consensus/src/consensus_observer/observer/consensus_observer.rs

```diff
@@ -416,13 +416,15 @@ impl ConsensusObserver {
 
         // Verify the block payload digests
         if let Err(error) = block_payload.verify_payload_digests() {
+            // Log the error and update the invalid message counter
             error!(
                 LogSchema::new(LogEntry::ConsensusObserver).message(&format!(
```

Comment:
I wonder if including some information about the peer might be helpful here

Reply:
+1 will peer_network_id help here to see who's sending bad messages?

Reply:
Aah, we can derive it from the existing logs, e.g., we only have 2 streams (we know the peers), and we only process the first message. But, let me add it to the logs anyway 😄